src/ResultBox.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    Typography
+} from '@material-ui/core';
+
+export default function ResultBox(props) {
+    return (<div>
+      <Dialog
+        maxWidth='xl'
+        open={props.open}
+        onClose={props.onClose}
+        scroll="paper"
+      >
+        <DialogTitle>{props.label}</DialogTitle>
+        <DialogContent dividers>
+            <Typography variant="h1" color="primary" style={{textAlign: "center"}}>
```

Comment:
Why not `DialogContentText`, like the other dialogs?

```suggestion
            <DialogContentText variant="h1" color="primary" style={{textAlign: "center"}}>
```